core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -913,6 +916,12 @@ object IO {
       )
     )
 
+  /**
+   * Yields execution to another thread.
+   */
+  final def `yield`: IO[Nothing, Unit] =
+    IO.sleep(0.seconds)
```

Comment:
This will unnecessary fork into scheduled pool. Why not 
```scala
IO.async { cb: Callback[Nothing, Unit] => cb(ExitResult.Completed(()) }
```
?

Reply:
The JVM `RTS` checks for zero duration sleeps and will submit to the regular pool instead: https://github.com/scalaz/scalaz-zio/blob/master/core/jvm/src/main/scala/scalaz/zio/RTS.scala#L88 . 

I'm not sure what this means for JS.

Reply:
In Javascript, it _should_ mean `setImmediate`